website/content/en/docs/contribution-guidelines/faq.md

```diff
@@ -0,0 +1,12 @@
+---
+title: FAQ
+weight: 200
+---
+
+#### Q: If find a problem, should I file a Pull Request or an Issue?
+
+A: We suggest that you start by filing an issue first because it can
+save you time if the change cannot be accepted for some reason.
```

Comment:
I'm not sure I understand "save you time if the change cannot be accepted.." why would an issue save them time? Are you thinking that the time they save is the time used to implement the PR?

Reply:
The idea is that if they file an issue, and we say that we cant accept the change they won't waste time on implementation.